themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss

```diff
@@ -0,0 +1,173 @@
+.paragraph--type--branch-contacts-info {
+  display: block;
+  padding: 20px;
+  padding-bottom: 0;
+  line-height: 1;
+
+  .field-branch-contacts-info {
+    .openy-branch-selector {
+      display: none;
+    }
+    .hb-branch-selector {
+      min-height: auto;
+      label {
+        margin-right: 5px;
+      }
+    }
+    .branch-header-row {
+      > div {
+        @include media-breakpoint-down(sm) {
+          margin-bottom: 20px;
+        }
+      }
+      align-items: center;
+      &div:last-child {
+        padding: 10px 0 0;
+      }
+      .back-link.fa,
+      .back-link a,
+      .field-location-phone i,
+      .header-address,
+      .camp-today-hour,
+      .wrapper-field-location-fax .field-location-fax a,
+      .wrapper-field-location-fax .field-location-fax a i,
+      .wrapper-field-location-phone .field-location-phone a,
+      .wrapper-field-location-phone .field-location-phone a i {
+      }
+      .back-link a {
+        font-size: 14px;
+        text-transform: none;
+      }
+      .hb-location-checkbox-wrapper {
+        @include media-breakpoint-down(sm) {
+          justify-content: start;
+        }
+        display: flex;
+        align-items: baseline;
+        justify-content: end;
+        color: $sky-blue;
+        label,
```

Comment:
Please, divide the CSS rules and selectors with an empty line -- to improve the code readability

Reply:
done